src/_common/content/components/media-upload/media-upload.ts

```diff
@@ -31,6 +31,14 @@ export default class AppContentMediaUpload extends Vue {
 	uploadProgress = 0;
 	uploadProcessing = false;
 
+	get placeholderMaxHeight() {
+		const maxHeight = this.owner.getContentRules().maxMediaHeight;
+		if (maxHeight !== null) {
```

Comment:
Yeah, this is checking explicitly that the value is not `null`. The other two examples is if you want to check that a value is truthy, usually for a binary decision, but it could be a value returned from an api or 3rd party package or something. So it could be false, 0, null, undefined, what have you. So you just go `!!value`.
In this case, we know it's either `null` or a numeric value, so we just check for `null`.

Reply:
sounds good, thanks for the explanation!